README.md

```diff
@@ -12,10 +12,9 @@ For the most comprehensive and up-to-date documentation on HabiticaMagicJS funct
 
 ## Limitations
 
-HabiticaMagicJS currently supports only `GET` requests at the moment, so it works best for projects that just need **read-only access to the Habitica API**.
+HabiticaMagicJS currently supports only `GET` and `POST` requests at the moment, so it works best for projects that just need **read-only access to the Habitica API**.
```

Comment:
This wording is confusing, as the addition of POST request functionality negates the limitation of read-only access. 

I'd suggest either removing the Limitations section or at least changing this sentence to better describe which functionality is still limited or in progress.